wao/notifier.py

```diff
@@ -1,6 +1,5 @@
 import sys
-sys.path.append("wao/")
```

Comment:
revert

Reply:
done

Reply:
revert

Reply:
revert file

Reply:
I am curious ... why do we need to append the wao path to system path here and not on the other files such as wao_strategy_controller_test...?